Assignment_2.md

```diff
@@ -0,0 +1,145 @@
+package Core_Java_Assignments;
+
+import java.util.Scanner;
+
+class calculator {
+
+    void circle() {
+        System.out.println("Please Enter Radious");
```

Comment:
Can you please handle the exception if the number value is not in the correct format?

Reply:
ok sir